packages/salesforcedx-vscode-replay-debugger/adaptertest/unit/adapter/apexReplayDebug.test.ts

```diff
@@ -853,6 +775,7 @@ describe('Replay debugger adapter - unit', () => {
         setValidLines.restore();
       });
 
+      /*JRS-these can be removed
```

Comment:
Why can these tests be removed? The code it was testing still exists in apexReplayDebug.ts. It was modified, so at the very least these tests should be modified. Otherwise, we’ve lost test coverage.

Reply:
Initially I thought these would be removed but when it turned out that the breakpoint info was needed in the adapter and the extension I'd forgotten to remove this comment. The tests aren't going to be removed, they'd just needed to be slightly tweaked.